public/css/schedule.less

```diff
@@ -67,6 +67,25 @@
   }
 }
 
+.timezone-warning {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  column-gap: 1em;
+
+  width: fit-content;
+  margin: 0 auto 1em auto;
+  padding: .5em 1em;
```

Comment:
you have a design rule, so please move the padding there, as padding generally is design related, without it, the box would look differently after all